### PR TITLE
gs: remove some pesky asserts

### PR DESCRIPTION
**Pull request by tadanokojin**
Really tired of my debugger crashing due to known behavior.

### CODE REVIEW

File: plugins/GSdx/GSState.cpp [hunk 1]
```diff
@@ -1265,11 +1265,6 @@ void GSState::GIFRegHandlerSCISSOR(const GIFReg* RESTRICT r)
 template <int i>
 void GSState::GIFRegHandlerALPHA(const GIFReg* RESTRICT r)
 {
-	ASSERT(r->ALPHA.A != 3);
```
Comment (lightningterror) [inline plugins/GSdx/GSState.cpp:1268]: Maybe replace them with a log instead?

Comment (tadanokojin) [reply plugins/GSdx/GSState.cpp:1268]: I don't really think the info is all that useful but I can stick a GL_REG in there if that makes you feel better.

Comment (lightningterror) [reply plugins/GSdx/GSState.cpp:1268]: Yeah it would be nice to have it.

Comment (tadanokojin) [reply plugins/GSdx/GSState.cpp:1268]: done.

**Pull request merged**